examples/argocd-with-applications/main.tf

```diff
@@ -35,13 +35,16 @@ module "network" {
 
 module "kubernetes" {
   depends_on = [module.network]
-  source     = "github.com/provectus/sak-kubernetes"
+  # source     = "github.com/provectus/sak-kubernetes"
+  # source     = "../../../sak-kubernetes-temporrary"
+  source     = "github.com/AlexSonar/sak-kubernetes"
```

Comment:
Could you change this back?

Reply:
I created forks of repositories [ "sak-kubernetes",  "sak-prometheus", ] temporarily,